packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -286,6 +261,82 @@
 				<tera-notebook-error v-if="!_.isEmpty(node.state?.errorMessage?.traceback)" v-bind="node.state.errorMessage" />
 			</tera-drilldown-section>
 		</template>
+
+		<template #sidebar-right>
+			<tera-slider-panel
+				v-model:is-open="isOutputSettingsPanelOpen"
+				direction="right"
+				header="Output Settings"
+				content-width="360px"
+			>
+				<template #overlay>
+					<tera-chart-settings-panel
+						:annotations="[]"
+						:active-settings="activeChartSettings"
+						@close="activeChartSettings = null"
+					/>
+				</template>
+				<template #content>
+					<div class="output-settings-panel">
+						<h5>Parameters</h5>
+						<tera-chart-settings-item
+							v-for="settings of chartSettings.filter(
+								(setting) => setting.type === ChartSettingType.DISTRIBUTION_COMPARISON
+							)"
+							:key="settings.id"
+							:settings="settings"
+							@open="activeChartSettings = settings"
+							@remove="removeChartSetting"
+						/>
+						<tera-chart-control
+							:chart-config="{ selectedRun: 'fixme', selectedVariable: selectedParameters }"
```

Comment:
I dont see `tera-chart-control` using the `selectedRun` field.
I understand we are not sending this field, but is what else is this fixme for?

Reply:
Hm.. not exactly sure. It looks like `fixeme` was used for temporary to make it type safe. `tera-chart-control` emits back the `selectedRun` value to the parent and I'm not exactly sure what other places use it. If this isn't used in any other places in our app. I think we should modify the tera-chart-control component itself to remove that field.

Reply:
This is from back the the days when we tried to support multiple runs per chart instead of singular runs, and the typing requires a selectedRun id. We labelled it as a fixme so readers know something is off.

@jryu01 as the notion of chart-config/chart-settings become more mature we should look to replace this.